products/cdk/get-started.adoc

```diff
@@ -23,7 +23,7 @@ Build your first container
 ## Step1 Windows Content
 The easiest way to install the Red Hat Container Development Kit is to use the Red Hat Development Suite installer.
 
-. Download the link:http://insert-link-here[Red Hat Development Suite 1.0] installer
+. Download the link:https://developers.redhat.com/download-manager/file/development-suite-1.0.0-GA-bundle-installer.exe[Red Hat Development Suite 1.0] installer
```

Comment:
This link actually needs to be https://developers.redhat.com/download-manager/file/cdk/development-suite-1.0.0-GA-bundle-installer.exe to get it to redirect to the right get-started page (this one) - can we get that fixed.